src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -118,6 +119,22 @@
   @Parameter(property = "dockerCmd")
   private String cmd;
 
+  /** The workdir for the image. Ignored if dockerDirectory is set */
+  @Parameter(property = "workdir")
+  private String workdir;
+
+  /** The user for the image. Ignored if dockerDirectory is set */
+  @Parameter(property = "user")
+  private String user;
+
+  /** 
+   * The run commands for the image. 
+   *  */
+  @Parameter(property = "dockerRuns")
```

Comment:
We did not used any special method to guarantee that. But we have never faced any issues with that. Run order from pom.xml was always kept unchanged.